cryptocurrency/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ghcr.io/deephaven/grpc-api
+COPY cryptoApp.app /
+COPY crypto.py /
```

Comment:
It's more efficient to combine these copies into a single COPY by putting the files into a subdirectory.

Also, while the root directory `/` *can* be configured as the application directory, it's probably a bad practice.

I'd suggest:

```dockerfile
COPY app.d/ /app.d
```

Reply:
And then, the property change: `-Ddeephaven.application.dir=/app.d`